src/core/blockchain/near/networks.ts

```diff
@@ -0,0 +1,27 @@
+import { IBlockchainNetwork } from '../types';
+
+// TODO: update this...
+export const networks: IBlockchainNetwork[] = [
```

Comment:
Shouldnt we display only one network - since there is no Mainnet for now?

Reply:
i changed the logic for getChainId selector. And with the new logic we should have at least one mainnet and one test net networks :). As we will have mainnet in future i don;t se a problem having this here.